### ISSUE TITLE
[BUG][MazPhoneNumberInput] Showing flag of country which is not in the onlyCountries property

### ISSUE THREAD

**Issue by elC0mpa** [LouisMazel/maz-ui#124]
**Describe the bug**
I am using a MazPhoneNumberImput component to to allow only United States country
```html
<MazPhoneNumberInput
                    :only-countries="['US']"
                    v-model="phoneNumberUS.value"
                    default-country-code="US"
                    @update="onUpdateUS"
                    :error="!phoneNumberUS.isValid"
                    :no-example="true"
                    :no-country-selector="cellView"
                  >
                  </MazPhoneNumberInput>
```

**Expected behavior**
It is expected that the PhoneInput will display United States flag and generate United States code. As you could see in the next section, this is not happening

**Screenshots**
![image](https://user-images.githubusercontent.com/53531665/101209436-7e5a5700-3652-11eb-8055-79a4d21c165b.png)

**Smartphone (please complete the following information):**
 - Device: Redmi 8
 - OS: MIUI 11 Android 10
 - Browser: Chrome
 - Version: 87.0.42

**Comment by LouisMazel** [comment]
Hi @elC0mpa ,

What number you use to init the component ? You must use the country code so that the component can recognize the country (+1 for US)

So, use the Prop `default-phone-number` and set the number like this:
- "8138478233"
- or "+18138478233"

**Not: "+8138478233"**

See this **[Code Sandbox Example](https://codesandbox.io/s/mazphonenumberinput-us-wnddc?file=/src/App.vue)**

FYI: 
- Use `default-phone-number` to set the init phone number
- `default-country-code` is the country code locale (US)
- `value` (v-model) is the result of both

I will release a fix to hide the flag when the country selector is disabled. Sorry for this.

**Comment by elC0mpa** [comment]
Thanks @LouisMazel !!!

**Comment by LouisMazel** [comment]
Solved in [v2.0.2](https://github.com/LouisMazel/maz-ui/releases/tag/v2.0.2)
